storage/rocksdb/rdb_converter.cc

```diff
@@ -0,0 +1,837 @@
+/*
+   Copyright (c) 2015, Facebook, Inc.
+
+   This program is f
+   i the terms of the GNU General Public License as published by
+   the Free Software Foundation; version 2 of the License.
+
+   This program is distributed in the hope that it will be useful,
+   but WITHOUT ANY WARRANTY; without even the implied warranty of
+   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+   GNU General Public License for more details.
+
+   You should have received a copy of the GNU General Public License
+   along with this program; if not, write to the Free Software
+   Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA  02111-1307  USA */
+
+/* This C++ file's header file */
+#include "./rdb_converter.h"
+
+/* Standard C++ header files */
+#include <algorithm>
+#include <map>
+#include <string>
+#include <vector>
+
+/* MySQL header files */
+#include "./log.h"
```

Comment:
In prep for 8.0 and in keeping with 5.7, we don't use "./xxx.h" idiom anymore for MySQL includes, just use "xxx.h" or more specific in 8.0 will be "sql/xxx.h".  see https://github.com/percona/percona-server/blob/50930fb400d882d432e9b2189fb95b3ed04356d3/storage/rocksdb/rdb_index_merge.h#L20 as an example.